packages/flutter/lib/src/widgets/app.dart

```diff
@@ -681,6 +682,48 @@ class WidgetsApp extends StatefulWidget {
   /// {@endtemplate}
   final bool debugShowCheckedModeBanner;
 
+  /// {@template flutter.widgets.widgetsApp.shortcuts}
+  /// The default map of keyboard shortcuts to intents for the application.
+  ///
+  /// By default, this is the output of [defaultShortcuts], called with
+  /// [defaultTargetPlatform]. If you wish to modify the default [shortcuts],
+  /// call [defaultShortcuts] and modify the resulting map, passing it as the
+  /// [shortcuts] for this app. Add to the bindings, or override specific
+  /// bindings for a widget subtree, by adding your own [Shortcuts] widget.
+  ///
+  /// See also:
+  ///
+  ///  - [LogicalKeySet], a set of [LogicalKeyboardKey]s that make up the keys
```

Comment:
Uber nit: here and below
```suggestion
  ///  * [LogicalKeySet], a set of [LogicalKeyboardKey]s that make up the keys
```